app/src/main/java/com/example/weatherapp/models/utils/DataFormatter.kt

```diff
@@ -39,7 +39,20 @@ fun Long.formatHour(resourceProvider: ResourceProvider, timeOffset: Int?): Strin
     timeOffset?.let {
         val zoneOffset = ZoneOffset.ofTotalSeconds(timeOffset)
         val dateTime = LocalDateTime.ofEpochSecond(this, 0, zoneOffset)
-        format = resourceProvider.getString(R.string.hour_format, dateTime.hour.toString())
+        if (dateTime.hour > 9) {
+            format = resourceProvider.getString(R.string.hour_format, dateTime.hour.toString())
+        } else {
+            format = resourceProvider.getString(
+                R.string.hour_format_single_digit,
+                dateTime.hour.toString()
+            )
+        }
     }
     return format
+}
+
+fun String.formatDescription(): String {
```

Comment:
Think of better name as it is visible globally. Hint: what this function is doing